contracts/GitcoinResolver.sol

```diff
@@ -14,6 +14,26 @@ import "./GitcoinAttester.sol";
  * @notice This contract is used to as a resolver contract for EAS schemas, and it will track the last attestation issued for a given recipient.
  */
 contract GitcoinResolver is ISchemaResolver {
+    mapping(address => bytes32) public passports;
+    mapping(bytes32 => address) public uuids;
+    uint256 private counter = 0;
+
+    function createUUID() public returns (bytes32) {
+        counter++;
+        return keccak256(abi.encodePacked(msg.sender, counter));
+    }
+
+    function setUUID() public {
```

Comment:
Instead of creating a new UID I think we should be utilizing the EAS UID that is passed as part of the attestation. [here](https://docs.attest.sh/docs/tutorials/resolver-contracts#expiration-time-resolver) is an example I found from their docs showing how to access values from the `Attestation` struct that used to make the attestation to EAS